specs/provisioning/provisioning.md

```diff
@@ -0,0 +1,29 @@
+# TFGrid provisioning
```

Comment:
I think this is the one

Reply:
yep I'll already merge this one so we get rid of the branch and can always create new PR to modify the flow if needed